experimental/PropertyDDS/packages/property-changeset/src/path_helper.js

```diff
@@ -332,6 +332,33 @@ PathHelper.quotePathSegment = function(in_pathSegment) {
   return '"' + in_pathSegment + '"';
 };
 
+/**
+ * Reverse a quoted/escaped string for a path seqment
+ *
+ * @param {string} in_quotedPathSegment   - The quoted/escaped path string to put in quotes
+ *
+ * @return {string} unquoted path string
+ */
+ PathHelper.unquotePathSegment = function(in_quotedPathSegment) {
+    if (!in_quotedPathSegment || typeof in_quotedPathSegment !== 'string') {
```

Comment:
When would this happen? The jsdoc says that you have to pass a string, so I would assume that the function should fail with an error if you provide a different type.

Reply:
The JSDoc is a copy from the original method. But we can either keep this logic here and update the JSDoc or move the logic to where this method is called if its necessary.